Assets/Scenes/Training_0.unity

```diff
@@ -215,6 +232,16 @@ PrefabInstance:
       value: 
       objectReference: {fileID: 11400000, guid: c753f81020ac40b4e96c0a53d1e0d8e8,
         type: 2}
+    - target: {fileID: 8908668329378533745, guid: e863cb15819a52b46a4983fad4a685b2,
+        type: 3}
+      propertyPath: trackingNumberOfGames
+      value: 2000
+      objectReference: {fileID: 0}
+    - target: {fileID: 8908668329378533745, guid: e863cb15819a52b46a4983fad4a685b2,
+        type: 3}
+      propertyPath: trackingPerformanceTF
+      value: 1
```

Comment:
I think it's probably best to have this turned off by default in the Training_0 scene. It looks like the prefab is set to have trackingPerformanceTF as false but this instance overrides it to true.

Reply:
Good idea - changed that and pushed it to this branch!